package.json

```diff
@@ -43,13 +43,16 @@
   "devDependencies": {
     "@changesets/cli": "^2.26.2",
     "@changesets/get-github-info": "^0.5.2",
-    "eslint": "^8.44.0",
+    "eslint": "^8.56.0",
     "prettier": "^2.8.4",
-    "typescript": "~5.1.6"
+    "typescript": "~5.4.2"
   },
   "peerDependencies": {
-    "eslint": ">=7",
-    "typescript": ">=4.5"
+    "eslint": ">=8.56.0",
+    "typescript": ">=4.7.5"
+  },
+  "engines": {
+    "node": ">=18.18.0"
   },
   "packageManager": "pnpm@8.6.6",
   "volta": {
```

Comment:
GitHub won't let me add a suggestion for this, but we should bump the volta Node version below to match at least the engines version above.